doc/source/changes/version_0_26.rst.inc

```diff
@@ -254,5 +254,20 @@ Fixes
 
 * fixed array creation with axis(es) given as string containing only one label (axis name and label were inversed).
 
+* fixed extracting LArray object from a CSV or Excel file when second axis is implicit (closes :issue:`372`).
+  For example, let say we want to read a CSV file 'pop.csv' with content ::
+
+  sex,2015,2016
```

Comment:
It would be more readable if you indented the table, as in:
```
sex, 2015, 2016
  M,   12,   10
  F,   11,   13
```
I am unsure whether or not we should keep the commas. I guess we should leave them in. See also comment above.